test/app-group.test.js

```diff
@@ -88,6 +88,7 @@ describe('application group', () => {
     const identifier = application.identifier;
     await application.deinit(DeinitSource.SignOut);
 
+    await Factory.sleep(0.0001);
```

Comment:
Context? Sleep calls in tests should either not be used, be self-evident, or have a comment describing why sleeping is important. Especially if this test passed before with no sleep.